docs/brainstorming/consolidation-strategy.md

```diff
@@ -0,0 +1,275 @@
+│  Purpose: Coordinate community efforts to accelerate HF neuroimaging     │
+│                                                                          │
+│  Current contents:                                                       │
+│  ├── scripts/                                                            │
+│  │   ├── download_ds004884.sh                                            │
+│  │   └── push_to_hub_ds004884_full.py                                    │
+│  ├── tools/                                                              │
+│  │   ├── arc-aphasia-bids/                                               │
+│  │   └── bids-neuroimaging/  (HF Space visualization)                    │
+│  └── README.md                                                           │
+└──────────────────────────────────────────────────────────────────────────┘
+                                    ▲
+                                    │ (consolidates)
+        ┌───────────────────────────┼───────────────────────────┐
+        │                           │                           │
+        ▼                           ▼                           ▼
+┌──────────────────┐  ┌──────────────────┐  ┌──────────────────────────┐
+│ arc-aphasia-bids │  │ stroke-deepisles │  │ huggingface/datasets     │
+│ (PRODUCTION)     │  │ -demo            │  │ (CONSUMPTION)            │
+│                  │  │                  │  │                          │
+│ • Upload ARC     │  │ • DeepISLES      │  │ • Pending PRs for        │
+│   to HF Hub      │  │   inference      │  │   BIDS loader, lazy      │
+│ • BIDS→HF        │  │ • Gradio UI      │  │   upload, DICOM          │
+│   conversion     │  │ • Full test      │  │ • Merged: Nifti,         │
+│ • Validation     │  │   suite          │  │   NiiVue viz             │
+└──────────────────┘  └──────────────────┘  └──────────────────────────┘
+                                │
+                                ▼
+                      ┌──────────────────┐
+                      │ DeepIsles        │
+                      │ (External Ref)   │
+                      │                  │
+                      │ • Nature 2025    │
+                      │ • Stroke lesion  │
+                      │   segmentation   │
+                      │ • Docker image   │
+                      └──────────────────┘
+```
+
+## Related Repositories
+
+| Repository | Type | Purpose | Link |
+|------------|------|---------|------|
+| neuroimaging-go-brrrr | Hub | Consolidation hub for neuroimaging tools | [GitHub](https://github.com/CloseChoice/neuroimaging-go-brrrr) |
+| arc-aphasia-bids | Production | Upload ARC dataset (ds004884) to HuggingFace Hub | [GitHub](https://github.com/The-Obstacle-Is-The-Way/arc-aphasia-bids) |
+| stroke-deepisles-demo | Production | DeepISLES inference demo with Gradio UI | [GitHub](https://github.com/The-Obstacle-Is-The-Way/stroke-deepisles-demo) |
```

Comment:
I would suggest not using Gradio, I did that already but somehow it never showed anything in there when using the gradio.HTML object. I assume they block external javascript.